doc/mysql.sql

```diff
@@ -50,7 +50,7 @@ CREATE TABLE builds (
     branch VARCHAR(255) DEFAULT 'master',
     cause VARCHAR(8191),
     internal INT, -- internal (project,class) build number, as opposed to global build number
-    status VARCHAR(32767) DEFAULT '[]',
+    status VARCHAR(8191) DEFAULT '[]',
```

Comment:
No. Change your max row size.
Otherwise I will have to create a table specific for `build_statuses` or create resource files.